assignments/array-methods.js

```diff
@@ -55,29 +55,47 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+let fullName = [first_name,last_name.foreach(currentValue => {
+
+console.log(fullName);})];
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+let allCaps = [(item,first_name) =>{
+    $;{item.first_name.UpperCase()}; 
+}];
+console.log(UpperCase); 
```

Comment:
```
console output: UpperCase is not defined
```

UpperCase is not a variable, it is a method. Also check your syntax here. 
syntax for these araay methods are: `Array.map(item => item.first_name.toUpperCase()`)